<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>league_edit_dialog</class>
 <widget class="QDialog" name="league_edit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>League Editor</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>681</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="league_list_widget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="league_edit_label">
          <property name="text">
           <string>Team Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="league_add_btn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="league_import_btn">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="league_export_btn">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="league_delete_btn">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="league_edit_btn">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
